pkg/remote/util.go

```diff
@@ -65,6 +69,40 @@ func GetImages() (string, string, string, string) {
 	return pfeImage + ":" + pfeTag, performanceImage + ":" + performanceTag, keycloakImage + ":" + keycloakTag, gatekeeperImage + ":" + gatekeeperTag
 }
 
+// Get kubeconfig
+func getKubeConfig() (*rest.Config, error) {
+	var config *rest.Config
+	var err error
+
+	// Use KUBECONFIG environment variable if set
+	kubeconfig, ok := os.LookupEnv("KUBECONFIG")
+	if ok && kubeconfig != "" {
+		// If multiple files provided choose first.
+		kubeconfig = strings.SplitN(kubeconfig, ":", -1)[0]
```

Comment:
I think you should use `filepath.SplitList()` - cwctl has to run on Windows uses a different separator character, SplitList will handle that: https://golang.org/src/path/filepath/path.go?s=5091:5127#L176